frameworks/PHP/ubiquity/.ubiquity/_swoole.php

```diff
@@ -1,18 +1,20 @@
 #!/usr/bin/env php
 <?php
 // swoole.php
-
-if (! defined ( 'DS' )) {
-	define ( 'DS', DIRECTORY_SEPARATOR );
-	define ( 'ROOT', __DIR__ . \DS .'..'.\DS. 'app' . \DS );
+if (! defined('DS')) {
+	define('DS', DIRECTORY_SEPARATOR);
+	define('ROOT', __DIR__ . \DS . '..' . \DS . 'app' . \DS);
 }
-$config=include ROOT.'config/config.php';
-$sConfig= include __DIR__.\DS.'swoole-config.php';
-$config["sessionName"]=null;
-$address=$sConfig['host'].':'.$sConfig['port'];
-$config ["siteUrl"] = 'http://'.$address;
+$config = include ROOT . 'config/config.php';
+$sConfig = include __DIR__ . \DS . 'swoole-config.php';
+$config["sessionName"] = null;
+$address = $sConfig['host'] . ':' . $sConfig['port'];
+$config["siteUrl"] = 'http://' . $address;
 require ROOT . './../vendor/autoload.php';
-$swooleServer=new \Ubiquity\servers\swoole\SwooleServer();
+$swooleServer = new \Ubiquity\servers\swoole\SwooleServer();
 $swooleServer->init($config, __DIR__);
-require ROOT.'config/swooleServices.php';
-$swooleServer->run($sConfig['host'],$sConfig['port']);
\ No newline at end of file
+require ROOT . 'config/swooleServices.php';
+$swooleServer->run($sConfig['host'], $sConfig['port'], [
+	'worker_num' => \swoole_cpu_num() * 2,
+	'reactor_num' => \swoole_cpu_num() * 2
```

Comment:
@jcheron, do you see any performance improvements with this or is it speculation?

Reply:
> @jcheron, do you see any performance improvements with this or is it speculation?

I think it's better, to be confirmed...

Reply:
If it is faster, then it is possible to have better coroutine implementation.
And theoretically will be slower in the plaintext and json tests.